### ISSUE TITLE
I am getting issues compiling my source code on ubuntu 22.X

### ISSUE THREAD

**Issue by lbhack** [frang75/nappgui_src#28]
hi @frang75 I am getting this is on ubuntu how I fix it :
usr/bin/ld: /home/parallels/nappgui_sdk/lib/gcc11_gtk3_x64/Debug/libsewer.a(bmath.cpp.o): undefined reference to symbol 'exp@@GLIBC_2.29'
/usr/bin/ld: /lib/x86_64-linux-gnu/libm.so.6: error adding symbols: DSO missing from command line
collect2: error: ld returned 1 exit status
make[2]: *** [demo/hello/CMakeFiles/HelloWorld.dir/build.make:111: demo/hello/Debug/HelloWorld] Error 1
make[1]: *** [CMakeFiles/Makefile2:358: demo/hello/CMakeFiles/HelloWorld.dir/all] Error 2
make: *** [Makefile:91: all] Error 2
![Screenshot 2022-12-20 at 8 05 08 PM](https://user-images.githubusercontent.com/2568488/208735596-8dd17cc1-718c-499a-af00-439139119795.png)

**Comment by frang75** [comment]
Hi @lbhack ! Thanks for the feedback.
This issue has been solved in NAppGUI 1.3 (will be released next 26/12/2022). 

For now, you can fix it adding these lines at the end of `targetLinkWithLibraries` function in `/prj/CMakeFunctions.txt`

```
# In GCC the g++ linker must be used
if (${CMAKE_SYSTEM_NAME} STREQUAL "Linux" OR ${CMAKE_SYSTEM_NAME} STREQUAL "Darwin")
    set_target_properties(${targetName} PROPERTIES LINKER_LANGUAGE CXX)
endif()
```

**Comment by frang75** [comment]
I close this issue because compile testing in Ubuntu 22.04 with NAppGUI 1.3 works fine. Also in other Ubuntu versions.
Please reopen if still you have problems with this.
